src/http.c

```diff
@@ -925,17 +983,71 @@ http_file_rewind(struct http_file *file)
 	file->offset = 0;
 }
 
-void
+size_t
+http_populate_cookies(struct http_request *req)
+{
+	int						 i, v;
+	size_t					 nlen;
+    struct http_cookie		*ck;
+    char					*c, *header, *value;
+    char					*cookies[HTTP_MAX_COOKIES];
+    const char				*semicol = ";";
+    const char				*comma = ",";
+    const char				*delim = NULL;
+
+    if (!http_request_header(req, "cookie", &c))
+	   return 0;
+
+    header = kore_strdup(c);
+    if (strstr(header, semicol) != NULL) {
+    	delim = semicol;
```

Comment:
Ok, agree.  Also, do we want to check name/value content for anything prohibited by RFCs? Also see http://stackoverflow.com/questions/1969232/allowed-characters-in-cookies